docs/en/observability/monitor-servers.asciidoc

```diff
@@ -32,3 +32,115 @@ This metric relies on the same indices as the logs.
 
 For information about which required fields the {metrics-app} uses to display host metrics, see the
 <<metrics-app-fields,Metrics field reference>>.
+
+[[enhanced-host-metrics]]
+== Enhanced host metrics
+
+Without leaving the <<view-infrastructure-metrics,*Inventory*>> page, you can view enhanced metrics relating to each host
+running in your infrastructure. On the waffle map, select the host to display the host metrics
+overlay.
+
+The enhanced metrics overlay contains the following tabs:
+
+// This is collapsed by default
+[%collapsible]
+.*Metrics*
+====
+
+[role="screenshot"]
+image::images/metrics-overlay.png[Host metrics]
+
+The *Metrics* tab displays CPU, load, memory, and network metrics relating to the host.
+You can change the time range to view metrics over the last 15 minutes, hour, 3 hours, 24 hours, or
+over the previous seven days. You can also hover over a specific time period on a chart to compare the various metrics
+at that given time.
+
+|=== 
+
+| *CPU* | Averages of `system.cpu.user.pct` and `system.cpu.system.pct`.
+
+| *Load* | Averages of `system.load.1`, `system.load.5`, and `system.load.15`.
+
+| *Memory* | For linux systems, memory used is the average of `system.memory.actual.used.bytes` and memory free is the average
+of `system.memory.actual.free`. For non-linux systems, memory used is the average of `system.memory.used.bytes` and memory free is the average
+of `system.memory.free`.
+
+| *Network* | Rates of `system.network.in.bytes` and `system.network.out.bytes`.
+
+|=== 
+====
+
+[%collapsible]
+.*Logs*
+====
+
+[role="screenshot"]
+image::images/logs-overlay.png[Host logs]
```

Comment:
Is event.dataset a default col? If not, I would remove it from the screen cap because my brain sees it and wants an explanation in the table that follows.

Reply:
it's a default.